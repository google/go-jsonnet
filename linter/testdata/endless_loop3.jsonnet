local x = x + 1; 42
