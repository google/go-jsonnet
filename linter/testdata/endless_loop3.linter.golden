testdata/endless_loop3:1:11-12 Endless loop in local definition

local x = x + 1; 42


